frontend/src/component/project/Project/ProjectInfo/ToggleTypesWidget.tsx

```diff
@@ -17,70 +18,65 @@ const StyledTypeCount = styled(StyledCount)(() => ({
-        useMemo(() => {
-            const release =
-                features?.filter(feature => feature.type === 'release')
-                    .length || 0;
-            const experiment =
-                features?.filter(feature => feature.type === 'experiment')
-                    .length || 0;
-            const operational =
-                features?.filter(feature => feature.type === 'operational')
-                    .length || 0;
-            const kill =
-                features?.filter(feature => feature.type === 'kill-switch')
-                    .length || 0;
-            const permission =
-                features?.filter(feature => feature.type === 'permission')
-                    .length || 0;
+interface IToggleTypeRowProps {
+    type: string;
+    Icon: OverridableComponent<SvgIconTypeMap>;
+    count: number;
+}
+const ToggleTypesRow = ({ type, Icon, count }: IToggleTypeRowProps) => {
+    const getTitleText = (str: string) => {
+        return str.charAt(0).toUpperCase() + str.slice(1).replace('-', ' ');
+    };
+    return (
+        <StyledParagraphGridRow data-loading>
+            <Icon fontSize="small" data-loading />
+            <div>{getTitleText(type)}</div>
+            <StyledTypeCount>{count}</StyledTypeCount>
+        </StyledParagraphGridRow>
+    );
+};
 
-            return {
-                release,
-                experiment,
-                operational,
-                kill,
-                permission,
-            };
-        }, [features]);
+export const ToggleTypesWidget = ({ features }: IToggleTypesWidgetProps) => {
+    const theme = useTheme();
+    const featureTypeStats = useMemo(() => {
+        const release =
+            features?.filter(feature => feature.type === 'release').length || 0;
+        const experiment =
+            features?.filter(feature => feature.type === 'experiment').length ||
+            0;
+        const operational =
+            features?.filter(feature => feature.type === 'operational')
+                .length || 0;
+        const kill =
+            features?.filter(feature => feature.type === 'kill-switch')
+                .length || 0;
+        const permission =
+            features?.filter(feature => feature.type === 'permission').length ||
+            0;
 
-    const ReleaseToggleIcon = getFeatureTypeIcons('release');
-    const ExperimentToggleIcon = getFeatureTypeIcons('experiment');
-    const OperationalToggleIcon = getFeatureTypeIcons('operational');
-    const KillToggleIcon = getFeatureTypeIcons('kill-switch');
-    const PermissionToggleIcon = getFeatureTypeIcons('permission');
+        return {
+            release,
+            experiment,
+            operational,
+            'kill-switch': kill,
+            permission,
+        };
+    }, [features]);
 
     return (
-        <StyledProjectInfoWidgetContainer>
+        <StyledProjectInfoWidgetContainer sx={{ padding: theme.spacing(3) }}>
```

Comment:
instead of `useTheme` you can do `{padding: theme => theme.spacing(3) }`

Reply:
Thanks, I didn't know that